source/common/quic/envoy_quic_proof_verifier_base.h

```diff
@@ -27,8 +27,6 @@ class EnvoyQuicProofVerifierBase : public quic::ProofVerifier,
   ~EnvoyQuicProofVerifierBase() override = default;
 
   // quic::ProofVerifier
-  // Return success if the certs chain is valid and signature of {
```

Comment:
why is the comment removed? can it no longer return failure?

Reply:
VerifyProof() is NOT_REACHED_GCOVR_EXCL_LINE now. So the comment is out dated.